src/UI/Layout.hs

```diff
@@ -25,6 +29,12 @@ offset by = wrap . Offset by
 resizeable :: (Size (Maybe a) -> Layout a b) -> Layout a b
 resizeable = wrap . Resizeable
 
+getMaxSize :: Layout a (Size (Maybe a))
+getMaxSize = wrap (Resizeable pure)
+
+measure :: Layout a b -> (Size a -> Layout a b) -> Layout a b
+measure child = wrap . Measure child
```

Comment:
Unfortunately we can’t do quite the same trick for `measure`, I think because of the child argument, tho I’m not entirely certain why.